src/index.js

```diff
@@ -1,51 +1,17 @@
 // import _ from 'lodash';
 import './style.css';
 
-// build array to store objects
+import Task from './add-remove.js';
 
-const arr = [];
-const elementLu = document.querySelector('.to-do-list');
+/*
+const takeData = document.querySelector('.input');
+// create button for submit
+const submitButton = document.querySelector('.submitButton');
```

Comment:
- [ ] Kindly remove all commented old pieces of code from your documents. This will ensure a clean code

Reply:
@Kamba56 i cant update my project. My hint linter is working wrong and i can solve it